### PR TITLE
You have been frozen by <name>

### PR DESCRIPTION
**Pull request by ghost**
Tags: Enchancment

### CODE REVIEW

**Comment by foxcqrn** [comment]
This is a single line change. As I recall, you have to change more than one line for the pull request to be accepted.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_freeze.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
         fd.setFrozen(!fd.isFrozen());
 
         msg(player.getName() + " has been " + (fd.isFrozen() ? "frozen" : "unfrozen") + ".");
-        msg(player, "You have been " + (fd.isFrozen() ? "frozen" : "unfrozen") + ".", ChatColor.AQUA);
+        msg(player, "You have been " + (fd.isFrozen() ? "frozen" : "unfrozen") + "by" + ChatColor.YELLOW + sender.getName() + ".", ChatColor.AQUA);
```
Comment (ghost) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_freeze.java:63]: Should be in AQUA to match with the rest of TFM.

**Comment by mayokek** [comment]
This does not meet the Contributing Guidelines.
- `PRs must not solely be grammar fixes, config-edits, small changes (less than 10 lines of code) or message rephrasing edits. Small edits such as these take very little effort to make and a lot more effort to review and integrate into the codebase.`
- `PRs must only address issues which are marked as "accepted".`

**Comment by lemonsked** [comment]
Where is the associated issue?

**Comment by JeromSar** [comment]
Needs an associated issue.

**Review by Wild1145 — CHANGES_REQUESTED** [review]
Please update the chat colour as mentioned in a line comment & add associated issue.

**Comment by Wild1145** [comment]
Thanks for contributing. I am closing this issue for inactivity. Should you have anything further to add in the future please either re-open the issue yourself or tag a member of staff and we will re-open the issue for further discussion.

**Pull request closed without merging**